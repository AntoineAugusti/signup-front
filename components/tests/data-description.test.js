import ReactTestRenderer from 'react-test-renderer'
import DataDescription from '../data-description'
import DGFIP_DATA_SET from '../../mock/data/dgfip'

describe('components | DataDescription', () => {
  describe('render', () => {
    const props = {
      dataset: DGFIP_DATA_SET
    }
    const renderer = ReactTestRenderer.create(<DataDescription {...props} />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
