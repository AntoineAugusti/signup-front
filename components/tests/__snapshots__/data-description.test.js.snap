// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | DataDescription render should match snapshot 1`] = `
<ul>
  <li>
    <h3 />
    <p />
    <a
      href={undefined}
      onClick={[Function]}
    >
      Essayez-moi !
    </a>
    <a
      href="/contractualisation"
      onClick={[Function]}
    >
      Ajouter à ma sélection de jeux de données !
    </a>
  </li>
</ul>
`;
